internal/backends/sqlite/metadata/registry.go

```diff
@@ -303,6 +310,10 @@ func (r *Registry) collectionCreate(ctx context.Context, dbName, collectionName
 		TableName: tableName,
 	}
 
+	if capped != nil {
+		r.colls[dbName][collectionName].Settings.Capped = capped
+	}
```

Comment:
```suggestion
	r.colls[dbName][collectionName].Settings.Capped = capped
```

Can be simplified right? Setting nil to non capped collection is ok I assume.